### PR TITLE
Use 'star' and 'star-border' icons instead of 'stars'.

### PR DESCRIPTION
**Pull request by maffoo**
@adamcw

### CODE REVIEW

File: client-js/app/elements/labrad-nodes.html [hunk 1]
```diff
@@ -73,10 +73,10 @@
       top: 5px;
     }
     paper-icon-button.autostart {
-      color: #75787B;
+      color: #85888B;
```
Comment (adamcw) [inline client-js/app/elements/labrad-nodes.html]: Change the colors back please, the grey doesn't match the other icons now.

File: client-js/app/elements/labrad-nodes.html [hunk 2]
```diff
@@ -73,10 +73,10 @@
...
     }
     paper-icon-button.autostartOn {
-      color: #0F9D58;
+      color: #0FBD78;
```
Comment (adamcw) [inline client-js/app/elements/labrad-nodes.html]: I chose a Google approved shade of green, and I preferred it.

File: client-js/app/elements/labrad-grapher.html [hunk 3]
```diff
@@ -123,8 +123,8 @@
       <div id="container">
         <paper-header-panel id="left-column">
           <div class="paper-header" id="buttons">
-            <paper-icon-button icon="stars" id="star" on-click="starClicked" title="show starred items"></paper-icon-button>
-            <paper-icon-button icon="delete" id="trash" on-click="trashClicked" title="show trashed items"></paper-icon-button>
+            <paper-icon-button icon="star" id="star" on-click="starClicked" title="Show only starred items."></paper-icon-button>
+            <paper-icon-button icon="delete" id="trash" on-click="trashClicked" title="Show trashed items."></paper-icon-button>
```
Comment (adamcw) [inline client-js/app/elements/labrad-grapher.html]: No other titles have full stops, but they are usually All Upper Title Case. I don't know if there is a style guide, but that's what I've been using.

**Comment by adamcw** [comment]
I like the other icons better.. but if you think this is an improvement I'm not THAT attached to them.

File: client-js/app/elements/labrad-nodes.html (hunk 1 above)
Comment (maffoo) [inline client-js/app/elements/labrad-nodes.html]: Ok, reverted. I found this and the green too dark for my taste, but it's just that, taste :-)

**Comment by maffoo** [comment]
I like having the star outline; I think it sets these off visually better from surrounding UI. But yes, this is all highly subjective...

**Comment by adamcw** [comment]
These have grown on me already so, LGTM.

**Pull request merged**